modules/planning/reference_line/reference_line_provider.cc

```diff
@@ -432,7 +432,9 @@ bool ReferenceLineProvider::GetReferenceLinesFromRelativeMap(
                        LaneWaypoint(lane_ptr, path_point.s())},
           path_point.kappa(), path_point.dkappa());
     }
-    reference_line->emplace_back(ref_points.begin(), ref_points.end());
+    ReferenceLine ref_line(ref_points);
```

Comment:
We can set priority after emplaced_back the reference line.

Reply:
ok